.github/workflows/test.yml

```diff
@@ -57,9 +57,11 @@ jobs:
                 pip install '.[test]'
 
         -   name: Test with pytest
+            if: ${{ matrix.os != 'macos-latest'}}
             run: |
                 python -m pytest --cov --cov-append --cov-report=term-missing --ignore=tests/integration/ -vv
         -   name: Upload coverage
+            if: ${{ matrix.os != 'macos-latest' }}
```

Comment:
Without this, will the test workflow just fail on mac then? I guess then it's only designed for GA and not to test on personal laptop installs, no? Why remove this? Because it's difficult on an M1?

Reply:
I was getting segmentation faults on GA that I didn't get  on my personal computer (M1 macbook). I also tried to rerun an already passing run on a different branch and that was suddenly throwing errors. It could be related to some updated dependency, but it's hard to fix the problem if it is machine specific.

Reply:
Am I reading this correctly that the tests are only run on non-mac systems via GA though? Your experience suggests it works only on macOS

Reply:
The tests were run on both Ubuntu and macOS, but only failed for macOS on GA. I couldn't reproduce the error on my personal macbook though and didn't want that issue to stall development